### PR TITLE
init json column. As start point)

### PR DESCRIPTION
**Pull request by vgordievskiy**

### CODE REVIEW

**Comment by vgordievskiy** [comment]
tested on postgresql.

File: lib/src/drivers/sql_driver.dart [hunk 1]
```diff
@@ -174,6 +174,7 @@ abstract class SqlDriver implements Driver {
   Object _serializeValue(Object value) {
     if (value is String || value is num) return value;
     if (value is DateTime) return value.toIso8601String();
+    if (value is Map) return JSON.encode(value);
```
Comment (emilniklas) [inline lib/src/drivers/sql_driver.dart]: How about including iterables here as well?

**Comment by emilniklas** [comment]
Good work by the way!

**Comment by vgordievskiy** [comment]
Thanks)

**Comment by emilniklas** [comment]
Nah man, thank you! I will publish a new version asap

**Comment by emilniklas** [comment]
`0.8.0` is up now

**Pull request merged**